kouncil-frontend/libs/common-login/src/lib/login/common-login.component.ts

```diff
@@ -0,0 +1,73 @@
+import {ChangeDetectionStrategy, Component, EventEmitter, Output} from '@angular/core';
+import {FormBuilder, FormControl, FormGroup, Validators} from "@angular/forms";
+import {User} from "./user";
+
+@Component({
+  selector: 'app-common-login',
+  template: `
+    <div class="icon-login-container">
+      <mat-icon aria-hidden="false" class="icon-login">person</mat-icon>
+    </div>
+    <div class="main-login">
+      <form (ngSubmit)="login()" class="login-form" [formGroup]="form">
+        <span class="login-info">Log in to your account</span>
+
+        <div #username class="login-field-container">
+          <div class="login-field-icon-container">
+            <mat-icon class="login-field-icon">person</mat-icon>
+          </div>
+          <div class="login-field-input-container">
+            <input placeholder="Login" class="input" matInput type="text" required
```

Comment:
https://stackoverflow.com/a/51716899
autocomplete="username" for login input
autocomplete="current-password" for  password input